### ISSUE TITLE
PC looses conection with smartphone

### ISSUE THREAD

**Issue by Katermerlin** [amanharwara/altus#196]
Since one of the Altus updates PC looses conection with smartphones, especially if there is a new version of Whatsapp...
Altus shows a message
https://www.pic-upload.de/view-37070863/Screenshot5.jpg.html
which tells me 
"Connecting with whatsapp
Ensure that your computer has an active internet connection"
It has indeed, but nothing happens, the message is permanent and doesnot change...

**Comment by amanharwara** [comment]
Is this on the latest version of Altus? You can find it out by clicking on Help > About.

**Comment by duiveldoder** [comment]
I have this issue since recently too on 4.5.6

**Comment by amanharwara** [comment]
> I have this issue since recently too on 4.5.6

If possible, update to the [latest version](https://github.com/amanharwara/altus/releases/tag/4.7.1)

**Comment by Ax3lA** [comment]
Hey Duiveldoder,

yes, get the latest version for sure.
A few weeks ago Amanharwara updated because of an issue where I commented with an other guy about the same issue.

Also try the ctrl+shift+r shortcut when the message appear to reload the webpage ;-)
At least that helps me.

@amanharwara hope you received the Latté grande 😄 
Thanks for that nice tool. I love it... (even I miss the sound functionality, but that could be an other ticket ;-)

Best regards
Alex

**Comment by amanharwara** [comment]
> hope you received the Latté grande smile
> Thanks for that nice tool. I love it... (even I miss the sound functionality, but that could be an other ticket ;-)

@Ax3lA yes I did receive it. thank you very much for that, i'm glad it's useful to you 😊 and yeah, definitely open a separate issue for the sound problem

**Comment by duiveldoder** [comment]
@Ax3lA 
> ctrl+shift+r

OMG thank you - that's the trick - every single problem I encounter is solved with this one.

Amanharwara you might consider adding a "button" for us users to do this.. with a "mouse over" info - refresh
